src/components/button/button.component.js

```diff
@@ -42,6 +44,16 @@ function renderChildren({
           </StyledButtonSubtext>
         )}
       </span>
+      {!children && iconType && iconPosition === "center" && (
```

Comment:
I think this should be 
```jsx
{iconType && (!children || iconPosition === "center") && ...
```

as if there's no children the icon should be positioned centre anyway

Reply:
I think we can get rid of `iconPosition` having a `center` option completely

Reply:
✅